druid/src/widget/widget_ext.rs

```diff
@@ -115,6 +115,13 @@ pub trait WidgetExt<T: Data>: Widget<T> + Sized + 'static {
         Container::new(self).background(brush)
     }
 
+    /// Wrap this widget in a [`Container`] with the provided `foreground`.
+    ///
+    /// See [`Container::background`] for more information.
```

Comment:
```suggestion
    /// Wrap this widget in a [`Container`] with the provided foreground `brush`.
    ///
    /// See [`Container::foreground`] for more information.
```

Also, please fix the above `background` method's docs to also say ```background `brush` ``` as there is no variable `background`.